docs/guides/redis/concepts/redis.md

```diff
@@ -304,16 +303,20 @@ KubeDB accept following fields to set in `spec.podTemplate:`
   - readinessProbe
   - lifecycle
 
-You can check out the full list [here](https://github.com/kmodules/offshoot-api/blob/ea366935d5bad69d7643906c7556923271592513/api/v1/types.go#L42-L259).
+You can check out the full list [here](https://github.com/kmodules/offshoot-api/blob/master/api/v2/types.go#L26C1-L279C1).
 Uses of some field of `spec.podTemplate` is described below,
 
-#### spec.podTemplate.spec.args
- `spec.podTemplate.spec.args` is an optional field. This can be used to provide additional arguments to database installation.
+#### spec.podTemplate.spec.tolerations
+
+The `spec.podTemplate.spec.tolerations` is an optional field. This can be used to specify the pod's tolerations.
+
+#### spec.podTemplate.spec.volumes
 
-### spec.podTemplate.spec.env
+The `spec.podTemplate.spec.volumes` is an optional field. This can be used to provide the list of volumes that can be mounted by containers belonging to the pod.
 
-`spec.podTemplate.spec.env` is an optional field that specifies the environment variables to pass to the Redis docker image.
+#### spec.podTemplate.spec.podPlacementPolicy
 
+`spec.podTemplate.spec.podPlacementPolicy` is an optional field. This can be used to provide the reference of the podPlacementPolicy
```

Comment:
This will be used by our `Petset controller` to place the db pods throughout the region, zone & nodes according to the policy. It utilizes kubernetes `affinity` & `podTopologySpreadContraints` feature to do so.